File: java/ErrorHandler.java
import java.util.Scanner;

public class ErrorHandler {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		while (scanner.hasNextLine()) {
			String line = scanner.nextLine();
			handleError(line);
		}
		scanner.close();
	}

	static void handleError(String error) {
		System.out.println("Handled " + error);
	}

}

File: python/error_handler.py
# usage: <other_command> 3>&1 1>&2 2>&3 | python error_handler.py

def main():
	while True:
		error = input()
		handle_error(error)


def handle_error(error):
	print("Handled: {}".format(error))


try:
	main()
except EOFError:
	pass
